controllers/global_search_setup.go

```diff
@@ -215,9 +215,11 @@ func logAndTrackError(errorList *[]error, err error, message string, keysAndValu
 func (r *SearchReconciler) enableGlobalSearch(ctx context.Context, instance *searchv1alpha1.Search) error {
 	errList := []error{} // Using this to allow partial errors and combine at the end.
 
+	mceInstallNS, err := r.getMCETargetNamespace(ctx)
+	logAndTrackError(&errList, err, "Failed to get mce installed namespace.")
 	// 1. Enable global search feature in the console.
 	// 1a.Add globalSearchFeatureFlag=true to configmap console-mce-config in multicluster-engine namespace.
-	err := r.updateConsoleConfig(ctx, true, "multicluster-engine", "console-mce-config")
+	err = r.updateConsoleConfig(ctx, true, mceInstallNS, "console-mce-config")
```

Comment:
In case of errors, maybe we should attempt using the default mce namespace.

Reply:
Thanks @jlpadilla , I think we can check the err and reset to use the default mce namespace to avoid the mceInstallNS is empty

Reply:
thanks!